index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Intuit Accounts Sign Up</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="icon" type="image/x-icon" href="https://accounts.intuit.com/images/favicons/favicon-16x16.png" />
+    <script src="main.js"></script>
+
+</head>
+
+<body>
+
+    <header class="header">
+
+        <div class="esc-link">
+            <p><small>Already have an Intuit account?<br /><a href="">I forgot my User ID or Password</a></small></p>
+        </div>
+
+        <button><i class="material-icons" style="font-size: 12px;">lock</i> Sign In</button>
+
+    </header>
+
+    <hr>
+
+    <section>
+
+        <div class="center">
+
+            <img src="https://lh3.googleusercontent.com/-A-E1hz61nqw/XDeMsm4F7zI/AAAAAAAAA5o/S_VcWrUSBx8x1bNu9WLI1DvJHj_tjfELQCK8BGAs/s512/logo.png"
+                alt="" width="150px" height="75px" style="opacity: 0.4">
+                
+            <ul id="icons-list">
+                <li><a href="#"><img src="https://accounts.intuit.com/IUS-Plugins/v2/images/mint-favicon.svg" alt=""
+                            width="25px" height="25px">
+                        <p>mint</p>
+                    </a></li>
+                <li><a href="#"><img src="https://accounts.intuit.com/IUS-Plugins/v2/images/quickbooks-favicon@2x.svg"
+                            alt="" width="25px" height="25px">
+                        <p>quickbooks</p>
+                    </a></li>
+                <li><a href="#"><img src="https://accounts.intuit.com/IUS-Plugins/v2/images/tt-favicon@2x.svg" alt=""
```

Comment:
The width and height property of the img tag is already in pixels. No need to specify units.